runtime/doc/vim_diff.txt

```diff
@@ -147,6 +146,15 @@ Highlight groups:
   |hl-TermCursorNC|
   |hl-Whitespace| highlights 'listchars' whitespace
 
+UI:
+		*E5408* *E5409* *g:Nvim_color_expr* *g:Nvim_color_cmdline*
+  UI now supports command-line coloring. Officially only |input()| and 
+  |inputdialog()| may be colored, temporary for testing purposes expressions 
+  (e.g. |i_CTRL-R_=|) and regular command-line (|:|) are colored by callbacks 
+  defined in `g:Nvim_color_expr` and `g:Nvim_color_cmdline` respectively. 
```

Comment:
`g:Nvim_` might not be a good convention because `sessionoptions+=globals` will try to store such variables if they are string/num types (doesn't apply here, but as a general convention ...).

Since these are temporary and deprecated, perhaps underscore prefix would make sense.